cmd/gcp-controller-manager/app/gke_approver.go

```diff
@@ -190,25 +193,29 @@ type csrValidator struct {
 var validators = []csrValidator{
 	{
 		name:       "kubelet client certificate with TPM attestation and SubjectAccessReview",
+		metricName: "kubelet-client-tpm",
```

Comment:
we are using both - and _ as delimiters in metric names here.

Reply:
Done